### ISSUE TITLE
New rule: no-absolute-path

### ISSUE THREAD

**Issue by jfmengels** [benmosher/eslint-plugin-import#530]
Based off [this comment](https://github.com/benmosher/eslint-plugin-import/issues/512#issuecomment-241761338)

This rule should forbid the use of imports with an absolute path. This is valid in Node.js, but should not appear in code in npm packages or in production, or better: never.

### Invalid

```js
import Foo from '/some/absolute/path.js';
```

### Valid

```js
import Foo from './some/relative/path.js';
import Foo from 'foo';
```

**Comment by benmosher** [comment]
I feel like I've already said this somewhere else (can't remember for sure), but what about `no-absolute-path(s)`?

**Comment by benmosher** [comment]
In the interest of pairing well with the plugin+`/`: `import/no-absolute-path`

**Comment by jfmengels** [comment]
:+1: for `import/no-absolute-path`
